### ISSUE TITLE
How to train using part of the raw datasets?

### ISSUE THREAD

**Issue by sadomss** [inventec-ai-center/bp-benchmark#9]
May I venture to ask if it is feasible for me to only use part of the dataset for data processing?
For example, if I want to only use the part_1 part of the UCI dataset for data processing and training, when using share_dataset.py for output, the generated csv file and mat file are significantly different from those when using the complete dataset.
Will using only a part of the dataset for data segmentation cause data leakage in the training set, validation set and test set?

**Comment by sergiogvz** [comment]
Sorry for the delayed response. It is possible to do so, but you would need to modify the code accordingly. I won't recommend it if the intention is to compare results with the benchmark paper. Besides, when using only one part of UCI, I would suggest to be careful and avoid shuffling the data before splitting. UCI dataset lacks subject identifiers, which makes hard to avoid leakage. This is the reason why we decided to use HOO with part 1 & 2 for training, 3 for validation, & 4 for test.

```  yaml   
# /code/process/core/config/splitting/uci_split.yaml & uci_feats_split.yaml
...
### Parameters for UCI to set the training, validation and test partitions related to the original splitting. 
tr_part: ['Part_1', 'Part_2'] 
val_part: ['Part_3'] 
ts_part: ['Part_4']  
```

**Comment by sadomss** [comment]
您好，我无法亲自回复您的邮件。我将尽快给您回复。
